src/metadata_inspector/__init__.py

```diff
@@ -120,8 +119,10 @@ def _get_files(input_: list[Path]) -> tuple[list[str], list[str]]:
         if not path:
             path = schema
         inp = Path(path).expanduser().absolute()
-        if schema in ("hsm", "slk"):
+        if schema in ("hsm", "slk") or inp.parts[1] == "arch":
             files_archive.append(str(inp))
+        if inp.exists() and inp.suffix in (".zarr",):
+            files_fs.append(str(inp))
         elif inp.is_dir() and inp.exists():
```

Comment:
What's the use case for folders? My experience with this branch of the code (admittedly with the broken Zarr-Thingie) is that it takes ages to run. Do you intend to promote this script to levante users? Because I don't see the use case for the web-databrowser.

Other than that, LGTM

Reply:
There is a cli version - and it's useful to get a combined view of datasets, for example different variables or timestamps combined in one dataset.